libs/community/langchain_community/chat_models/tongyi.py

```diff
@@ -117,11 +148,25 @@ def convert_message_chunk_to_message(message_chunk: BaseMessageChunk) -> BaseMes
     if isinstance(message_chunk, HumanMessageChunk):
         return HumanMessage(content=message_chunk.content)
     elif isinstance(message_chunk, AIMessageChunk):
-        return AIMessage(content=message_chunk.content)
+        # assert message_chunk is None
+        return (
+            AIMessage(
+                content=message_chunk.content,
+                tool_calls=message_chunk.additional_kwargs["tool_calls"],
```

Comment:
this is confusing, because in `convert_dict_to_message` we are parsing tool calls (e.g., in call to `parse_tool_call`) but here we are passing them in although they are already parsed.

at this point, is the following true?
```python
if message_chunk.additional_kwargs["tool_calls"]:
    item = message_chunk.additional_kwargs["tool_calls"][0]

assert isinstance(item["args"], dict)  # not a string
```

Reply:
Sorry for delayed response, having a really tough time with my job 😫 
This was a mistake, I've rewritten the method referring langchain_openai's `base` module.